CHANGES.txt

```diff
@@ -11,6 +11,7 @@ Relevant changes in behaviour compared to 2.0b1:
   coordinates (fix compatibility with Shapely 1.8).
 - Raise ValueError for non-finite distance in the ``buffer()`` and ``offset_curve()``
   methods on the Geometry classes (consistent with Shapely 1.8).
+- Add `directed` parameter to `line_merge` for GEOS >= 3.11.0
```

Comment:
We can maybe add it in this list instead: https://shapely.readthedocs.io/en/latest/release/2.x.html#bindings-for-new-geos-functionalities (that is the detailed changelog for the final 2.0 release)

Reply:
Updated the guide, let me know if the change makes sense. It doesn't really fit into the existing groups.